src/components/pages/dashboard/dashboard.js

```diff
@@ -317,6 +318,12 @@ export class Dashboard extends Component {
         ? deviceIds.length - onlineDeviceCount
         : undefined;
 
+    // Add parameters to Time Series Insights Url
+    const timeSeriesParamUrl =
+      timeSeriesExplorerUrl
+        ? timeSeriesExplorerUrl + '&relativeMillis=1800000&timeSeriesDefinitions=[{"name":"Devices","splitBy":"iothub-connection-device-id"}]'
```

Comment:
See other comment about string concatenation.

Also, if these URL are complicated to build and similar to each other, would a helper function be useful?

Reply:
Opting to leave in as is. If we end up adding more complicated TSI queries we can revisit in a shared function in models perhaps.